README.md

```diff
@@ -1 +1,40 @@
-# NewyorkTimes
\ No newline at end of file
+# NewyorkTimes
+Project 1: The positioning and floating elements project as described in the Odin project.
+
+![screenshot](images\NYTproj.PNG)
+
+Semantic HTML is used for the mark-up of this project. Elements are positioned using flexbox as well as floats.
+
+## Bult With
+
+- HTML 5
+- CSS 3
+
+## Live Demo
+
+[Live Demo Link](https://blakbox23.github.io/NewyorkTimes/)
+
+## Authors
+- GitHub: [@Blakbox23](https://github.com/blakbox23)
+- Twitter: [@blakbox23](https://twitter.com/blakbox23)
+- LinkedIn: [Peter Mbuthia](https://www.linkedin.com/in/peter-mbuthia-b15791182/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](lic.url) licensed.
```

Comment:
[required] This license link is broken, you can fix it by adding a real license link, or you can just remove it instead of put the one which is broken